### ISSUE TITLE
jubagraph: create_node returns the node id is not unique after loading the model data

### ISSUE THREAD

**Issue by kazuki** [jubatus/jubatus#1149]
In jubagraph, create_node returns the node id is not unique after loading the model data.

How to reproduce this bug

1. Launch jubagraph with default config
```
$ jubagraph -f /usr/share/jubatus/example/config/graph/default.json
```
2. Create node and edge.
```
$ python
>>> import jubatus
>>> from jubatus.graph.types import Edge
>>> c = jubatus.graph.client.Graph('127.0.0.1', 9199, '')
>>> c.create_node()
'0'
>>> c.create_node()
'1'
>>> c.create_edge('0', Edge({}, '0', '1'))
2
>>> c.update_node('0', {'foo': 'bar'})
True
>>> c.get_node('0')
node{property: {'foo': 'bar'}, in_edges: [], out_edges: [2]}
>>> c.save('hoge')
{'xxx.xxx.xxx.xxx_9199': '/tmp/xxx.xxx.xxx.xxx_9199_graph_hoge.jubatus'}
```
3. Restart jubagraph
4. Load model and create node
```
$ python
>>> import jubatus
>>> from jubatus.graph.types import Edge
>>> c = jubatus.graph.client.Graph('127.0.0.1', 9199, '')
>>> c.load('hoge')
>>> c.get_node('0')
node{property: {'foo': 'bar'}, in_edges: [], out_edges: [2]}
>>> c.create_node()
'0'  <== wrong. expected node id is '3' (0,1 is node, 2 is edge)
>>> c.create_edge('0', Edge({}, '0', '1'))
1  <== wrong. expected edge id is 4 (0,1,3 is node, 2 is edge)
```

jubagraph outputs
```
2016-09-29 14:31:22,153 26605 INFO  [server_util.cpp:414] starting jubagraph 0.9.4 RPC server at xxx.xxx.xxx.xxx:9199
    pid                  : 26605
    user                 : kazuki
    mode                 : standalone mode
    timeout              : 10
    thread               : 2
    datadir              : /tmp
    logdir               : 
    log config           : 
    zookeeper            : 
    name                 : 
    interval sec         : 16
    interval count       : 512
    zookeeper timeout    : 10
    interconnect timeout : 10

2016-09-29 14:31:22,154 26605 INFO  [server_util.cpp:165] load config from local file: /usr/share/jubatus/example/config/graph/default.json
2016-09-29 14:31:22,154 26605 INFO  [graph_serv.cpp:155] config loaded: {
  "parameter" : {
    "damping_factor" : 0.9,
    "landmark_num" : 5
  },
  "method" : "graph_wo_index"
}

2016-09-29 14:31:22,154 26605 INFO  [server_helper.hpp:226] start listening at port 9199
2016-09-29 14:31:22,154 26605 INFO  [server_helper.hpp:233] jubagraph RPC server startup
2016-09-29 14:31:43,609 26608 DEBUG [graph_serv.cpp:226] node created: 0
2016-09-29 14:31:44,768 26607 DEBUG [graph_serv.cpp:226] node created: 1
2016-09-29 14:31:49,248 26608 DEBUG [graph_serv.cpp:339] edge created: 2 ( 0 => 1 )
2016-09-29 14:31:53,552 26608 DEBUG [graph_serv.cpp:236] node updated: 0
2016-09-29 14:32:01,840 26607 INFO  [server_base.cpp:142] starting save to /tmp/xxx.xxx.xxx.xxx_9199_graph_hoge.jubatus
2016-09-29 14:32:01,840 26607 INFO  [server_base.cpp:185] saved to /tmp/xxx.xxx.xxx.xxx_9199_graph_hoge.jubatus
```

after restart
```
2016-09-29 14:32:06,359 26695 INFO  [server_util.cpp:414] starting jubagraph 0.9.4 RPC server at xxx.xxx.xxx.xxx:9199
    pid                  : 26695
    user                 : kazuki
    mode                 : standalone mode
    timeout              : 10
    thread               : 2
    datadir              : /tmp
    logdir               : 
    log config           : 
    zookeeper            : 
    name                 : 
    interval sec         : 16
    interval count       : 512
    zookeeper timeout    : 10
    interconnect timeout : 10

2016-09-29 14:32:06,360 26695 INFO  [server_util.cpp:165] load config from local file: /usr/share/jubatus/example/config/graph/default.json
2016-09-29 14:32:06,360 26695 INFO  [graph_serv.cpp:155] config loaded: {
  "parameter" : {
    "damping_factor" : 0.9,
    "landmark_num" : 5
  },
  "method" : "graph_wo_index"
}

2016-09-29 14:32:06,360 26695 INFO  [server_helper.hpp:226] start listening at port 9199
2016-09-29 14:32:06,361 26695 INFO  [server_helper.hpp:233] jubagraph RPC server startup
2016-09-29 14:32:23,697 26698 INFO  [server_base.cpp:60] starting load from /tmp/xxx.xxx.xxx.xxx_9199_graph_hoge.jubatus
2016-09-29 14:32:23,697 26698 INFO  [server_base.cpp:82] loaded from /tmp/xxx.xxx.xxx.xxx_9199_graph_hoge.jubatus
2016-09-29 14:32:31,168 26697 DEBUG [graph_serv.cpp:226] node created: 0
2016-09-29 14:32:35,392 26698 DEBUG [graph_serv.cpp:339] edge created: 1 ( 0 => 1 )
```

**Comment by kmaehashi** [comment]
Refs #613, #661

We need ``reset_id_generator`` function as in ``anomaly_serv``.

**Comment by kmaehashi** [comment]
Fixed via #1156
